### PR TITLE
Added instruction of how to invite a bot

### PR DESCRIPTION
**Pull request by Commandserver**
- completed the example of how to create a bot application

### CODE REVIEW

**Comment by Commandserver** [comment]
btw should i use `“` or `"`?

**Comment by SirObby** [comment]
In what context?

**Comment by SirObby** [comment]
Just use `"`'s

**Comment by Commandserver** [comment]
> Just use `"`'s

ok

File: docpages/INDEX.md [hunk 1]
```diff
@@ -35,12 +35,10 @@ The library may work fine in other operating systems too, if you run a D++ bot o
 
 ## Getting started
 * [GitHub Repository](https://github.com/brainboxdotcc/DPP)
-* [Discord Server](https://discord.gg/dpp)
```
Comment (braindigitalis) [inline docpages/INDEX.md:38]: I like the PR but I don't think this change below is needed?
it just shortens the links which harms SEO

**Review by braindigitalis — CHANGES_REQUESTED** [review]

**Comment by Commandserver** [comment]
okay. changed back

**Review by braindigitalis — APPROVED** [review]

**Pull request merged**